src/librawspeed/tiff/TiffEntry.h

```diff
@@ -78,7 +78,7 @@ class TiffEntry
   ushort16 getShort(uint32 num=0) const;
   short16 getSShort(uint32 num=0) const;
   float getFloat(uint32 num=0) const;
-  std::string getString() const;
+  std::string getString(bool trimmed = true) const;
```

Comment:
Okay, so i think this is the only thing in this pr i don't quite agree with.
Trimming the strings from exif/makernotes by default seems dangerous.